manifests/kustomize/third-party/metacontroller/base/crd.yaml

```diff
@@ -1,45 +1,533 @@
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
+  annotations:
+    "api-approved.kubernetes.io": "unapproved, request not yet submitted"
```

Comment:
what's this?

Reply:
This is probably pending for sigs approval.